Domain.Sql/EfDbConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Data.Entity;
+using System.Data.Entity.SqlServer;
+
+namespace Microsoft.Its.Domain.Sql
+{
+    public class EfDbConfiguration : DbConfiguration
+    {

```

Comment:
This name isn't really adding information, since DbConfiguration is already defined in EF. SqlClientConfiguration would be more accurate but there's a larger question here. The inheritance approach is not composable, so defining this in Its.Domain.Sql is of limited value. And since you only applied this to one DbContext, it implies that it may not be needed at all. Can you explain more about your thinking on this?